ui/src/views/Documentation/components/Warning/index.jsx

```diff
@@ -0,0 +1,8 @@
+import React from 'react';
+import ErrorPanel from '../../../../components/ErrorPanel';
+
+const Warning = ({ msg, keepOpen }) => (
+  <ErrorPanel warning error={msg} onClose={keepOpen} />
+);
```

Comment:
Here's some untested code that I think will allow the error panel to start accepting the message as a child string.

```jsx
import React from 'react';
import ErrorPanel from '../../../../components/ErrorPanel';
import { string } from 'prop-types';

const Warning = ({ children }) => (
  <ErrorPanel warning error={children} onClose={null} />
);

Warning.propTypes = {
  children: string.isRequired,
};
```

Reply:
Thanks for sharing.
I'm glad that my solution is identical to suggested by you. :-) It works perfectly.